pyqt-apps/siriushla/as_ps_control/detail_widget/auxiliary_intlk_data.py

```diff
@@ -0,0 +1,63 @@
+"""Temporary auxiliary interlock data module."""
```

Comment:
is it not time for the window to get these labels from the correponding label PVs? I think having to maintain both IOC csdev and HLA synchronized with PS  BSMP spec is not appropriate. At least HLA cound get it from constants in siriuspy. The idea of having such label PV is to avoid over-maintanance.

Reply:
I don't know about getting from the PV, but It could get from `siriuspy`, couldn't it?

Reply:
@fernandohds564 now, I get from siriuspy.
This solution was necessary because this new labels for Q1...Q4 are in a separated branch  that will be merged only after all FAPs firmware are updated

Reply:
In this case I agree with @anacso17, @xresende. Since it is only a temporary solution, I think it is better to create these labels here than compromise the execution time or depend on the connection with a PV to open a window.

Reply:
In a previous version, this window took the information from PV. 
We decided to change it to avoid timeout problems and to be possible to see Interlock labels in disconnection cases...

Reply:
yes, yes. I misread. I though it was a general function for building widget labels for interlocks.

Reply:
@xresende  yes! In the final version, the interlock labels will be consulted in `siriuspy`!